dts/bindings/serial/nordic,nrf-uart-common.yaml

```diff
@@ -7,73 +7,88 @@ properties:
     interrupts:
       required: true
 
+    disable-rx:
+      type: boolean
+      description: |
+        Disable UART reception capabilities.
+
+    current-speed:
+      description: |
+        Initial baud rate setting for UART. Only a fixed set of baud
+        rates are selectable on these devices.
+      enum:
+        - 1200
+        - 2400
+        - 4800
+        - 9600
+        - 14400
+        - 19200
+        - 28800
+        - 31250
+        - 38400
+        - 56000
+        - 57600
+        - 76800
+        - 115200
+        - 230400
+        - 250000
+        - 460800
+        - 921600
+        - 1000000
+
     tx-pin:
       type: int
-      required: true
       description: |
-        The TX pin to use.
+        DEPRECATED: This option will only be used if the new pin control driver
+        is not enabled.
 
+        The TX pin to use.
         For pins P0.0 through P0.31, use the pin number. For example,
         to use P0.16 for TX, set:
-
             tx-pin = <16>;
-
         For pins P1.0 through P1.31, add 32 to the pin number. For
         example, to use P1.2 for TX, set:
-
             tx-pin = <34>;  /* 32 + 2 */
```

Comment:
In https://github.com/zephyrproject-rtos/zephyr/pull/39429#discussion_r749174625 I meant these three removed empty lines, too. Removing them affects appearance of the bindings documentation, so I think they should be also restored.

Reply:
sorry, I misunderstood that part, fixed